client/me/purchases/manage-purchase/index.jsx

```diff
@@ -215,7 +215,7 @@ const ManagePurchase = React.createClass( {
 			upgradesActions.addItem( redemptionItem );
 		}
 
-		page( '/checkout/' + this.props.selectedSite.slug );
+		page( '/checkout/' + this.props.selectedSite.domain );
```

Comment:
To have one consistent way of accessing it. Should I change the other way around to `.slug`. I think we were using `.domain` in Domain Management section.